code/online_challenges/src/hackerrank/Compare-the-triplets

```diff
@@ -0,0 +1,23 @@
+#include<iostream>
```

Comment:
1. Add a `.cpp` file extension and call the file `compare_the_triplets.cpp`. Move it to its own directory as well.